cdk/test/rule-manager.test.ts

```diff
@@ -0,0 +1,21 @@
+import { expect as expectCDK, haveResource } from '@aws-cdk/assert';
+import * as cdk from '@aws-cdk/core';
+import * as Cdk from '../lib/rule-manager/rule-manager';
+
+test('SQS Queue Created', () => {
```

Comment:
Presume these won't pass at present, we could replace these with a snapshot test re: @akash1810's recommendation, again v. happy to take a look Monday

Reply:
I will add some detail into here.

Reply:
This is from the CDK template. I would recommend removing them and then implementing our testing in a separate PR. Happy to include it in this PR if you feel strongly.